src/test/git-elegant-feature.bats

```diff
@@ -8,9 +8,34 @@ setup() {
     fake-pass git "fetch --tags"
     fake-pass git pull
     fake-pass git "checkout -b test-feature"
+    fake-pass git "stash save elegant-git"
+    fake-pass git "stash apply stash^{/elegant-git}"
+    fake-pass git "stash drop stash@{0}"
 }
 
-@test "exit code is 0 when run 'git-elegant feature'" {
+@test "exit code is 0 when run 'git-elegant feature test-feature'" {
+  run git-elegant feature test-feature
+  [ "$status" -eq 0 ]
+}
+
+@test "exit code is 255 when run 'git-elegant feature'" {
+  run git-elegant feature
+  [ "$status" -eq 255 ]
+}
+
+@test "print message when run 'git-elegant feature'" {
+  run git-elegant feature
+  [[ "${lines[0]}" =~ "Feature name is not set" ]]
+}
+
+@test "exit code is 0 when run 'git-elegant feature' with changes" {
+  fake-pass git status "Saved working directory"
```

Comment:
@ivan-podorozhnyi I think you fake-pass the wrong command, is it? The same about line#38.

Reply:
Also, may a good idea is to `fake-fail` `git stash drop ..` with a specific exit code and check that this code appears in the end. it will guarantee that unstash logic was invoked.

Reply:
> I think you fake-pass the wrong command, is it? The same about line#38.

@extsoft Sorry, my bad. Resolved.

> Also, may a good idea is to fake-fail git stash drop .. with a specific exit code and check that this code appears in the end. it will guarantee that unstash logic was invoked.

@extsoft I added a unit test to check it. Please, take a look.

But another thing that I found is that in case of merge conflicts our app closes without dropping stashed changes. Do you think it's normal behavior? Should we apply some changes or add a separate issue to discuss it?

Reply:
> But another thing that I found is that in case of merge conflicts our app closes without dropping stashed changes.

@ivan-podorozhnyi the idea is to store changes for a customer. If we can apply them, we haven't delete.